packages/trader/src/Modules/Reports/Components/reports-meta.jsx

```diff
@@ -1,12 +1,16 @@
 import classNames from 'classnames';
 import React from 'react';
 
-const ReportsMeta = ({ i18n_heading, i18n_message, filter_component, className }) => (
+const ReportsMeta = ({ i18n_heading, i18n_message, filter_component, className, optional_component }) => (
     <div className={classNames('reports__meta', className)}>
-        <div className='reports__meta-description'>
-            <h1 className='reports__meta-description--heading'>{i18n_heading}</h1>
-            <p className='reports__meta-description--paragraph'>{i18n_message}</p>
-        </div>
+        {i18n_heading && (
```

Comment:
Do we want to check for both `i18n_heading` and `i18n_message` here? Perhaps there might be a heading with no description?

Reply:
We have called this component only 3 times and all have passed both props together. Since, it is not reasonable that one passes only one of them I wrote it like this, but if you think this might happen, I will apply changes.

Reply:
I think it's fine then, thank you.